akka-http-core/src/main/java/akka/http/javadsl/model/AttributeKeys.java

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright (C) 2019-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.http.javadsl.model;
+
+public final class AttributeKeys {
+    public static final AttributeKey<RemoteAddress> remoteAddress = (AttributeKey<RemoteAddress>)(Object)akka.http.scaladsl.model.AttributeKeys.remoteAddress();
```

Comment:
I think if you import `akka.http.impl.util.JavaMapping._` you can use `akka.http.scaladsl.model.AttributeKeys.remoteAddress().asJava` instead of having to do the cast

Reply:
This is a java class and I cannot find  where `JavaMaping` used in java.

Reply:
On the other hand, `AttributeKey[T]` has a type parameter, I'm not sure `JavaMapping` can convert this kind of type.

Reply:
Yeah, I see. `JavaMapping` could handle that, but I agree it's probably not worth moving this class from Java to Scala just for that.

The cast is safe (since the scala AttributeKey subclasses the java AttributeKey and the scala RemoteAddress subclasses the Java RemoteAddress), and I don't really see a neater way to write it, so this might be fine.